pkg/network/setup/netstat.go

```diff
@@ -90,60 +90,71 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		return nil
 	}
 
-	multusStatusNetworksByName := netvmispec.IndexInterfaceStatusByName(
+	oldIfaceStatusesByName := netvmispec.IndexInterfaceStatusByName(
 		vmi.Status.Interfaces,
 		func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
-			return netvmispec.ContainsInfoSource(ifaceStatus.InfoSource, netvmispec.InfoSourceMultusStatus)
+			return ifaceStatus.PodInterfaceName != "" ||
+				netvmispec.ContainsInfoSource(ifaceStatus.InfoSource, netvmispec.InfoSourceMultusStatus)
 		},
 	)
 	vmiInterfacesSpecByName := netvmispec.IndexInterfaceSpecByName(vmi.Spec.Domain.Devices.Interfaces)
 
-	interfacesStatus := ifacesStatusFromDomainInterfaces(domain.Spec.Devices.Interfaces)
-	interfacesStatus = append(interfacesStatus,
+	updatedIfaceStatuses := ifacesStatusFromDomainInterfaces(domain.Spec.Devices.Interfaces)
```

Comment:
nit: `new`,`fresh` and `current` are also nice alternatives for the prefix.
But I do not think you should have changed it. It makes the review harder and it adds no real value IMO.

Reply:
This change was reverted.